.github/workflows/ci.yml

```diff
@@ -60,6 +60,22 @@ jobs:
       - uses: actions-rs/cargo@4ff6ec2846f6e7217c1a9b0b503506665f134c4b
         with:
           command: test
+      - uses: actions-rs/cargo@4ff6ec2846f6e7217c1a9b0b503506665f134c4b
+        with:
+          command: run
+          args: -- --model cuboid --export cuboid.3mf
+      - uses: actions-rs/cargo@4ff6ec2846f6e7217c1a9b0b503506665f134c4b
+        with:
+          command: run
+          args: -- --model group --export group.3mf
+      - uses: actions-rs/cargo@4ff6ec2846f6e7217c1a9b0b503506665f134c4b
+        with:
+          command: run
+          args: -- --model spacer --export spacer.3mf
+      - uses: actions-rs/cargo@4ff6ec2846f6e7217c1a9b0b503506665f134c4b
+        with:
+          command: run
+          args: -- --model star --export star.3mf
```

Comment:
Could we maybe automate this by adding a script to the repository that iterates over all models in `models/`, running each? This is fine for now, but executing all models automatically would be more robust going forward, as models are added, removed, or renamed.

Such a script would have to run on all the platforms we run CI builds on. There are many ways to do this. [Nushell](https://www.nushell.sh/) has been suggested for this purpose in the past.